dts/bindings/gpio/altr,pio-1.0.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022 Intel Corporation
```

Comment:
Wrt naming, why "altr" and why "pio-1.0"?

Is it named intentionally this way to be compatible with some pre-existing vendor / Linux DT binding?

Otherwise, I would suggest "altera,gpio-pio-1.0"

Reply:
It is suggested ([discussion_r1070692791](https://github.com/zephyrproject-rtos/zephyr/pull/53766#discussion_r1070692791))  to match with the upstream binding for the same IP: https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/gpio/gpio-altera.txt